realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsUnmanagedTests.java

```diff
@@ -109,9 +86,11 @@ public void copyToAndFromRealm() {
         assertEquals(1, parents.size());
         assertEquals(parent, parents.first());
 
+        // backlinks fields in unmanaged object created from managed object are null.
+
         unmanagedChild = realm.copyFromRealm(child);
         assertEquals(unmanagedChild.getFieldId(), 1);
-        assertEquals(new AllJavaTypes().getObjectParents(), unmanagedChild.getObjectParents());
+        assertNull(unmanagedChild.getObjectParents());
 
         RealmResults<AllJavaTypes> queryResults = realm.where(AllJavaTypes.class).equalTo("fieldId", 1).findAll();
```

Comment:
@bmeike I was not thinking that a backlinks field of unmanaged object can return non-null object.
As you said in other comment, users can keep non-null RealmResults object to backlink field of unmanaged objects.

I'll revert this change.